### ISSUE TITLE
Set up a demo to present the playerclass component

### ISSUE THREAD

**Issue by jackcasey067** [uchicago-cs/chiventure#1098]
We need a demo to present for the final presentation.  We should show off as many features that we worked on this quarter as we can.

**Comment by jackcasey067** [comment]
I set up the branch `playerclass/demo` to contain the changes for this issue.  I also added the file `2021-demo.c` to work in.  It can be built and run via the command:
```
$ make examples && src/playerclass/examples/2021-demo 
```

**Comment by farm-zaq** [comment]
Multiclass_demo() has been implemented, although there are a few incorrect outputs that need to be fixed.

**Comment by jackcasey067** [comment]
The four mini demo functions technically work, and they are good enough, although I would prefer for them to be a bit more fleshed out maybe.  That being said, each mini demo should be quick to run through (~2 minutes).

Regardless of where the demo is, I am opening a PR for it at 11:30 or so to beat the soft freeze.

**Comment by ideclue** [comment]
The `demo_item_interactions` function has been expounded upon to create a short, but effective display of class-item interactions (including restrictions and multipliers). However, the WDL Demo is currently causing a segmentation fault and may need to be removed for the presentation.

**Comment by jackcasey067** [comment]
Closing Comment:
We determined that running the executable in the wrong directory was causing the segfault (the program was unable to find the example WDL file it loads).  I updated the code so that this mistake is caught and prints a helpful error message instead of segfaulting.

It is important that the program is run from the `build/` is to run the command: 
`make examples && src/playerclass/examples/2021-demo`, which builds and runs the file.

Having solved this problem and having made a Pull Request, this issue is now closed.
